docs/asciidoc/modules/ROOT/pages/database-integration/vectordb/weaviate.adoc

```diff
@@ -250,8 +272,17 @@ CALL apoc.ml.rag(paths, $attributes, $question, $confPrompt) YIELD value
 RETURN value
 ----
 
+which returns the result as string. For example, "The city where foo equals one is Berlin."
```

Comment:
Come sopra, puoi anche togliere il `For example, "The city ...`, tanto l'esempio completo sta già sulla pagina `xref::ml/rag.adoc` che sta linkata